test/object-store/sync/flx_sync.cpp

```diff
@@ -614,7 +602,7 @@ TEST_CASE("flx: client reset", "[sync][flx][client reset][baas]") {
 
     SECTION("Recover: offline writes and subscription (single subscription)") {
         config_local.sync_config->client_resync_mode = ClientResyncMode::Recover;
-        auto&& [reset_future, reset_handler] = make_client_reset_handler();
+        auto&& [reset_future, reset_handler] = reset_utils::make_client_reset_handler();
```

Comment:
or just `using namespace reset_utils` at the top of the file.

Reply:
I kept the namespace for these calls, since nearly all of the other places where `reset_utils` functions are used include the namespace (e.g. see test/object-store/sync/client_reset.cpp)

Reply:
right. my suggestion was so you don't have to make these changes.